org.aimodels.aiy.json

```diff
@@ -0,0 +1,48 @@
+{
+  "app-id": "org.aimodels.aiy",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "aiy",
+  "finish-args": [
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "aiy",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/visioninit/aiy.git",
+          "branch": "main"
+        }
+      ],
+      "modules": [
+        "resources/build/flatpak/pypi-dependencies.json",
+        {
+          "name": "other-modules"
+        }
+      ],
+      "build-commands": [
+        "pip3 install pyinstaller",
+        "pyinstaller --onefile aiy.py --name aiy --distpath dist/.",
+        "pip-licenses --with-system --with-authors --with-url > dist/licenses.txt",
+        "cat LICENSE dist/licenses.txt > dist/all-licenses.txt",
+        "install -D dist/aiy /app/bin/aiy",
+        "cp dist/all-licenses.txt /app/bin/LICENSE",
+        "install -D resources/build/flatpak/org.aimodels.aiy.metainfo.xml /app/share/metainfo/org.aimodels.aiy.metainfo.xml",
+        "install -D resources/build/flatpak/org.aimodels.aiy.desktop /app/share/applications/org.aimodels.aiy.desktop",
+        "install -D resources/build/flatpak/org.aimodels.aiy.png /app/share/icons/hicolor/128x128/apps/org.aimodels.aiy.png"
+      ],
+      "cleanup": [
+        "venv",
+        ".egg-info",
+        ".whl",
+        "dist",
+        "build",
+        "files/lib/python3.10"
+      ]
```

Comment:
I'm pretty this cleanup rule doesn't do what you think it does. there is no `files/lib/python3.10`

Reply:
After the app is built, I wanted to remove the unneeded files. Should I do this in the build commands?

Reply:
Only files installed are kept, and cleanup are applied to them.


`files/lib/python3.10` is likely wrong you probably want `/lib/python3.10`. the rest look like build artifect.
But then it's a python app, are you sure you want to remove the python modules.

Reply:
The python app is built to a binary with pyinstaller, so the python bits aren't still required. I tried to initially set it up to use the python interpreter, however there was some issue with pygments that I wasn't able to resolve. I noticed that pygments was used as part of the base image, and after reviewing with others, tried the pyinstaller method.